### ISSUE TITLE
NTR: IL-15 high in blood (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8429]
New term request IL-15 high in blood (elthat_190811040629): IL15; Interleukin 15  UMLS: 5195545

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000439
- label: Increased circulating interleukin 15 concentration
- parents: [Abnormal circulating interleukin concentration (HP:0030782)](https://hpo.jax.org/app/browse/term/HP:0030782)
- pmid: PMID:22064066
- def: The concentration of interleukin 15 (IL-15) in the blood circulation is above the upper limit of normal.
- comment: Interleukin-15 (IL-15) is a pleiotropic cytokine with a broad range of biological functions in many diverse cell types.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8429
- synonyms: IL-15 high in blood(NONE)
